external/fv3fit/fv3fit/pytorch/graph.py

```diff
@@ -0,0 +1,183 @@
+import torch
+import tensorflow as tf
+import numpy as np
+import dgl
+import dataclasses
+from .._shared.training_config import Hyperparameters
+from toolz.functoolz import curry
+from fv3fit.pytorch.graph_predict import PytorchModel
+from fv3fit.pytorch.graph_builder import build_graph, GraphConfig
+from fv3fit.pytorch.graph_config import GraphNetwork, GraphNetworkConfig
+from fv3fit.pytorch.graph_loss import LossConfig
+from fv3fit.pytorch.graph_optim import OptimizerConfig
```

Comment:
I still think you should put graph functions under a `graph` submodule to avoid this duplication of a prefix in module names.

Are LossConfig and OptimizerConfig really graph-specific, or just specific to pytorch? If they're not graph-specific they don't need graph-specific naming for their module.

Reply:
Looking into it these don't appear to be specific to graphs, so these modules should be renamed.

Reply:
Do you mean a submodule under pytorch? You're right the loos and optimization are not graph-specific. I'll change them